### ISSUE TITLE
Discord link has expired 

### ISSUE THREAD

**Issue by sokrypton** [bytedance/Protenix#37]

**Comment by zhangyuxuann** [comment]
Hi @sokrypton , thank you for your kind reminder. The previous link was only valid for 7 days. We have updated the link of the disorder in the README.md. This new [discord link](https://discord.com/invite/8ZMWy89aMf) has no expiration date.

**Comment by Linmj-Judy** [comment]
I tried to join discord group but failed. Is this link still work?
![image](https://github.com/user-attachments/assets/f64a5416-ad9d-4e29-8724-50ffc04b483d)

**Comment by zhangyuxuann** [comment]
@Linmj-Judy yes, the [link](https://discord.com/invite/8ZMWy89aMf) works. I have asked some colleagues to verify it.
